### PR TITLE
refactor: test utils

### PR DESCRIPTION
**Pull request by renlulu**

### CODE REVIEW

File: tests/claim.zil.test.ts [hunk 1]
```diff
@@ -69,166 +62,175 @@ beforeAll(async () => {
...
-        "IncreaseAllowance",
-        globalStakingContractAddress,
-        Number(100000000000000).toString()
+    const tx0: any = await zilliqa.contracts
+        .at(globalToken0ContractAddress)
+        .call(
+          "IncreaseAllowance",
+          getJSONParams({
+            spender: ["ByStr20", globalStakingContractAddress],
+            amount: ["Uint128", Number(100000000000000).toString()]
```
Comment (ghost) [inline tests/claim.zil.test.ts]: `Number(100000000000000).toString()` can be `100000000000000`

File: tests/claim.zil.test.ts [hunk 2]
```diff
@@ -69,166 +62,175 @@ beforeAll(async () => {
...
+      .new(fs.readFileSync(CONTRACTS.token2.path).toString(), init2)
       .deploy(TX_PARAMS, 33, 1000, true);
-      globalToken2ContractAddress = contract.address;
+    globalToken2ContractAddress = contract.address;
 
+    const init3 = getJSONParams({
+      _scilla_version: ["Uint32", 0],
+      init_contract_owner: ["ByStr20", getTestAddr(OWNER)],
+      init_staking_token_address: ["ByStr20", globalToken0ContractAddress],
+      blocks_per_cycle: ["Uint256", "10"],
```
Comment (ghost) [inline tests/claim.zil.test.ts]: `"10"` can be `10`

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
LGTM

File: tests/claim.zil.test.ts (hunk 2 above)
Comment (noel-yoo) [reply tests/claim.zil.test.ts:121]: it's ok to change this in the next PR.

**Pull request merged**